notebooks/growth_model_labelling.ipynb

```diff
@@ -0,0 +1,1812 @@
+{
```

Comment:
<p>In the first paragraph, the 2 lines are repeated. Can you just remove the last 2 lines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>